libs/model/src/policies/ContestWorker.policy.ts

```diff
@@ -31,9 +48,83 @@ export function ContestWorker(): Policy<typeof inputs> {
           payload.community_id,
           payload.thread_id,
         );
+
```

Comment:
should we create another query for this? similar to GetActiveConstestmanagers above?

Reply:
will apply on next pass